src/__tests__/createGetDataElementValue.test.js

```diff
@@ -363,7 +370,40 @@ describe('function returned by createGetDataElementValue', function () {
         );
         var value = getDataElementValue('testDataElement');
 
-        expect(value).toBe('');
+        expect(value).toBe(dataElementValue);
+      }
+    );
+
+    it(
```

Comment:
This test ensures that the `defaultValue` is respected when it is available and the call to grab the value of a dataElement doesn't return a result.